sky/templates/runpod-ray.yml.j2

```
@@ -0,0 +1,110 @@
+cluster_name: {{cluster_name}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.skylet.providers.runpod.RunPodNodeProvider
+  region: {{region}}
+
```

This is to avoid our code to consider the same node with changed `launch_config`
```suggestion
    disable_launch_config_check: true

```